### PR TITLE
Public documentation for running a Scroll node

### PR DESCRIPTION
**Pull request by isabellewei**
## Description

guide on how to run a Scroll node

other changes

- modified the `Aside` component so that it can display no title

### CODE REVIEW

File: src/content/docs/en/developers/guides/running-a-scroll-node.mdx [hunk 1]
```diff
@@ -0,0 +1,406 @@
...
+    $ cd l2geth-source
+    $ git checkout VERSION
+    ```    
+2. Ensure that you are using version 1.18 of `go`.
+3. Install the gcc compiler: `sudo apt install build-essential`.
+4. Build `l2geth`: `make nccc_geth`. The binary is now in `build/bin/geth`.
+5. Define a command alias: `alias l2geth=./build/bin/geth`.
+
+{/* <Aside>
+`nccc` stands for “*no circuit capacity checker*”. This feature disables an expensive block validation rule that is not crucial on follower nodes. We recommend that you do not enable it. **If you do, please let us know so that we can notify you in case of breaking ccc upgrades.**
```
Comment (isabellewei) [inline src/content/docs/en/developers/guides/running-a-scroll-node.mdx]: I commented this out, do we feel like it should be included?

Comment (dghelm) [reply src/content/docs/en/developers/guides/running-a-scroll-node.mdx]: I don't think we need to include it.

**Comment by dghelm** [comment]
As mentioned in conversation, I'd love to see an intro to this clarifying who the audience is, since this is a "guide" and not just a technical reference. Maybe mention that many developers will use the public RPC and then advance to managed infrastructure by RPC providers in the ecosystem. Also to make clear that this is a "follower node" that can participate in the public mempool of transactions, but does not participate in consensus or block building.

Lastly, can we have subheadings in troubleshooting that labels each error? It kind of runs together as it displays now.

File: src/content/docs/en/developers/guides/running-a-scroll-node.mdx [hunk 2]
```diff
@@ -0,0 +1,405 @@
...
+### Download and Build L2geth
+
+1. Clone the latest version of `l2geth`. 
+
+    ```bash
+    $ git clone https://github.com/scroll-tech/go-ethereum l2geth-source
+    $ cd l2geth-source
+    $ git checkout VERSION
+    ```    
+2. Ensure that you are using version 1.18 of `go`.
```
Comment (Thegaram) [inline src/content/docs/en/developers/guides/running-a-scroll-node.mdx]:
```suggestion
2. Ensure that you are using version 1.20 of `go`.
```

I'll update the other doc

File: src/content/docs/en/developers/guides/running-a-scroll-node.mdx [hunk 3]
```diff
@@ -0,0 +1,405 @@
...
+
+---
+
+## Build and Run Using Docker
+
+<Aside type="caution" title="Warning">
+Running the node in Docker might have a significant impact on node performance.
+</Aside>
+
+### Download L2geth
```
Comment (Thegaram) [inline src/content/docs/en/developers/guides/running-a-scroll-node.mdx]:
```suggestion
### Build L2geth Image
```

File: src/content/docs/en/developers/guides/running-a-scroll-node.mdx [hunk 4]
```diff
@@ -0,0 +1,405 @@
...
+
+1. Build the container using `make mockccc_docker`. This will produce the image `scrolltech/l2geth:latest`.
```
Comment (Thegaram) [inline src/content/docs/en/developers/guides/running-a-scroll-node.mdx]:
```suggestion
1. Build the image using `make mockccc_docker`. This will produce the image `scrolltech/l2geth:latest`.
```

File: src/content/docs/en/developers/guides/running-a-scroll-node.mdx [hunk 5]
```diff
@@ -0,0 +1,405 @@
...
+    
+
+### Run L2geth
+
+1. Run the `l2geth` container using the name `l2geth-docker`:
+    
+    ```bash
+    $ docker run --rm --name l2geth-docker -v $(pwd):/volume -p 8545:8545 -e RUST_LOG="info" -e CHAIN_ID="534352" scrolltech/l2geth:latest \
+        --scroll \
+    		--datadir "/volume/l2geth-datadir" \
```
Comment (Thegaram) [inline src/content/docs/en/developers/guides/running-a-scroll-node.mdx]:
```suggestion
        --datadir "/volume/l2geth-datadir" \
```

File: src/content/docs/en/developers/guides/running-a-scroll-node.mdx [hunk 6]
```diff
@@ -0,0 +1,405 @@
...
+    
+    Alternatively, you can download the `l2geth` Docker image using `docker pull scrolltech/l2geth:VERSION`. In this case, substitute this image name in the commands below.
+    
+
+### Run L2geth
+
+1. Run the `l2geth` container using the name `l2geth-docker`:
+    
+    ```bash
+    $ docker run --rm --name l2geth-docker -v $(pwd):/volume -p 8545:8545 -e RUST_LOG="info" -e CHAIN_ID="534352" scrolltech/l2geth:latest \
```
Comment (Thegaram) [inline src/content/docs/en/developers/guides/running-a-scroll-node.mdx:114]: Actually, does it break without `CHAIN_ID="534352"`?

File: src/content/docs/en/developers/guides/running-a-scroll-node.mdx [hunk 7]
```diff
@@ -0,0 +1,405 @@
...
+    "number": "0x0",
+    "gasUsed": "0x0",
+    "parentHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+    "baseFeePerGas": null
+    }
+  ```
+</ToggleElement>
+
+---
+## Troubleshooting
```
Comment (Thegaram) [inline src/content/docs/en/developers/guides/running-a-scroll-node.mdx]: I think the first two could be removed, they seem to be irrelevant now

Comment (isabellewei) [reply src/content/docs/en/developers/guides/running-a-scroll-node.mdx]: I'm just going to remove the entire Troubleshooting section then

**Review by dghelm — APPROVED** [review]
